hexa/pipelines/models.py

```diff
@@ -262,6 +267,7 @@ def upload_new_version(
             description=description,
             external_link=external_link,
             zipfile=zipfile,
+            config=config if config else self.config,
```

Comment:
`Pipeline.config` is only used for the former system of pipelines using Airflow. For our implementation we can ignore that.

Reply:
Are you sure ? It's in the `upload_new_version` method that is on `Pipeline`